Crashlytics/Crashlytics/Controllers/FIRCLSReportManager.m

```diff
@@ -522,7 +522,20 @@ - (void)crashReportingSetupCompleted:(FIRCLSProfileMark)mark {
   });
 }
 
-- (BOOL)checkBundleIDExists {
+- (BOOL)validateAppIdentifiers {
+  // When the ApplicationIdentifierModel fails to initialize, it is usually due to
+  // failing computeExecutableInfo. This can happen if the user sets the
+  // Exported Symbols File in Build Settings, and leaves off the one symbol
+  // that Crashlytics needs, "__mh_execute_header" defined as _MH_EXECUTE_SYM
+  // From https://github.com/firebase/firebase-ios-sdk/issues/5020
+  if (!self.appIDModel) {
+    FIRCLSErrorLog(
+        @"Crashlytics could not find the main file symbol, and cannot start up. This can happen "
+        @"when Exported Symbols File is set in Build Settings. To resolve this, add "
+        @"\"__mh_execute_header\" as a newline to your Exported Symbols File.");
```

Comment:
The message seems inconsistent with the comment.

Reading the comment, I would think I need to add:
`__mh_execute_header=_MH_EXECUTE_SYM`

Reading the message:
`__mh_execute_header`

Reply:
I'll update the comment - should be just `__mh_execute_header`